source/crud/configure.txt

```diff
@@ -191,6 +191,28 @@ The following example sets the read preference to ``ReadPreference.Secondary`` f
 For more information about read preference, see :manual:`Read Preference
 </core/read-preference/>` in the {+mdb-server+} manual.
 
+Retryable Reads and Writes
+--------------------------
+
+The {+driver-short+} automatically retries certain read and write operations a single time
+if they fail due to a network or server error.
+
+You can explicitly disable retryable reads or retryable writes by setting the ``RetryReads``
+or ``RetryWrites`` options on a ``MongoClientSettings`` object and passing this object to the
+``MongoClient`` constructor. The following example disables retryable reads and writes for
+a client:
+
+.. literalinclude:: /includes/fundamentals/code-examples/ReplicaSetConfigs.cs
+   :start-after: start-retry-reads-writes
+   :end-before: end-retry-reads-writes
+   :emphasize-lines: 2-3
+   :language: csharp
+   :dedent:
+
+To learn more about supported retryable read and retryable write operations, see
```

Comment:
Not sure if it would be worth mentioning here or if there is a more appropriate place, but should also say that those can be set already in the connection string?